analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/SymbolicContextTest.cs

```diff
@@ -93,6 +94,34 @@ public void SetOperationConstraint_WithNewValue_CreatesNewProgramState()
             result[operation].HasConstraint(DummyConstraint.Dummy).Should().BeTrue();
         }
 
+        [TestMethod]
+        public void SetSymbolConstraint_WithExistringValue()
```

Comment:
Would be great to update the name as well to describe the expected behavior.

Reply:
Should have same destiny as conversion above